donate.html

```diff
@@ -150,22 +150,17 @@ <h1>Our Story</h1>
   <!-- Donor Promise -->
   <div class="donate-promise">
     <div class="container">
-      <!-- Header -->
-        <div class="promise-header">
-          <div class="row">
-            <div class="col-md-6"></div>
-            <div class="col-md-6"><h1>Our Donor Promise</h1></div>
-          </div>
-        </div>
-
-      <!-- Body -->
+      <!-- Image -->
       <div class="row">
         <div class="col-md-6">
           <div class="promise-image">
-              <img src="img/2017_img.jpg" class="center-block" width="350" height="200">
+              <img src="img/2017_img.jpg" class="center-block">
           </div>
         </div>
         <div class="col-md-6">
```

Comment:
so when the columns collapse vertically in mobile view, we want the header and text to appear before the image. the way to reorder columns based on breakpoints can be done through here: https://getbootstrap.com/docs/4.0/layout/grid/#reordering. but basically to save you time, it would be something like adding "order-1 order-md-2" to this column which says that this column will appear second until the website reaches the medium breakpoint, then this column will appear first. the same logic can be applied to the column above.